src/bo4e/enum/anrede.py

```diff
@@ -1,13 +1,14 @@
-from enum import Enum
-
+"""
+Übersicht möglicher Anreden, z.B. eines Geschäftspartners.
+"""
 
-class Anrede(str, Enum):
-    """
-    Übersicht möglicher Anreden, z.B. eines Geschäftspartners.
-    """
+from enum import Enum
 
-    HERR = "HERR"
-    FRAU = "FRAU"
-    EHELEUTE = "EHELEUTE"
-    FIRMA = "FIRMA"
-    INDIVIDUELL = "INDIVIDUELL"  # Individuell festgelegt
+_anrede = {
+    "HERR": "HERR",
+    "FRAU": "FRAU",
+    "EHELEUTE": "EHELEUTE",
+    "FIRMA": "FIRMA",
+    "INDIVIDUELL": "INDIVIDUELL"
+}
+Anrede = Enum("Anrede", _anrede)
```

Comment:
Das Enum-Thema hat sich hier versehentlich in den PR gemogelt.
Alle Dateien die im enum-Ordner liegen wurden so geändert, dass nur ein Dicitonary erzeugt wird, was letztlich in ein Enum umgewandelt wird.
Dazu hatte ich auch eine [Frage auf SO](https://stackoverflow.com/questions/65339635/how-to-deserialise-enumeration-with-string-representation) gestellt.